boot/bootutil/include/bootutil/bootutil_log.h

```diff
@@ -74,34 +74,58 @@ extern "C" {
 #define BOOT_LOG_LEVEL_INFO	3
 #define BOOT_LOG_LEVEL_DEBUG	4
 
+/*
+ * The compiled log level determines the minimum level that can be
+ * printed.  Messages at or below this level can be printed, provided
+ * they are also enabled through the Rust logging system, such as by
+ * setting RUST_LOG to bootsim::api=info.
+ */
 #ifndef BOOT_LOG_LEVEL
-#define BOOT_LOG_LEVEL BOOT_LOG_LEVEL_ERROR
+#define BOOT_LOG_LEVEL BOOT_LOG_LEVEL_INFO
```

Comment:
I think it belongs with this patch, since if the default level isn't changed, the runtime logging level won't do anything.

I do need to remove the defines that occur in other places, or at least make the defaults more reasonable for on-target.